README.md

```diff
@@ -26,7 +45,7 @@ What does the following selector do?  `ul.dropdown > li`?
 Select 1:
 ```
 [] Selects all li's which are directly inside a ul of class dropdown (children)
-[] Selects all li's which are anywhere inside a ul of class dropdown (any descendant)
+[X] Selects all li's which are anywhere inside a ul of class dropdown (any descendant)
```

Comment:
Close, but it's actually the first option. e.g. `div > p` will select all paragraphs which are immediate children of a div, while a space (`div p`) will select paragraphs in a div, regardless of whether they are immediate children, or grandchildren, etc